src/mgmt/cli_mgmt.c

```diff
@@ -1567,16 +1567,16 @@ dc_mgmt_init()
 	int		rc;
 	uint32_t        ver_array[2] = {DAOS_MGMT_VERSION - 1, DAOS_MGMT_VERSION};
 
-	rc = daos_rpc_proto_query(mgmt_proto_fmt_v2.cpf_base, ver_array, 2, &dc_mgmt_proto_version);
+	rc = daos_rpc_proto_query(mgmt_proto_fmt_v3.cpf_base, ver_array, 2, &dc_mgmt_proto_version);
```

Comment:
Are there compatibility issues with removing support for v2? Or was it already not used?

Reply:
Not used because we only maintain compatibility for last and current versions which is 3 & 4 now DAOS_MGMT_VERSION is 4.